integreat_cms/cms/forms/regions/region_form.py

```diff
@@ -139,6 +141,40 @@ def clean(self):
         else:
             cleaned_data["matomo_id"] = None
 
+        # Validate custom prefix
+        if cleaned_data.get("custom_prefix"):
+            # Get the administrative divisions as conflicting options
+            administrative_divisions = [
+                label
+                for choice, label in self.fields["administrative_division"].choices
+            ]
+            if self.instance.default_language:
+                # If region already has a default language, translate the labels
+                with override(self.instance.default_language.slug):
```

Comment:
We could say that we check if the string is in any existing translation of prefixes. But that is probably not a case that will happen that often.

Reply:
@svenseeberg I changed the validation method to check for all existing translation. Can you test again? Thanks!

Reply:
Works as expected. Thanks again :-)